### PR TITLE
report: make tools menu focus-able

### LINKED ISSUE #9168: Tools menu steals tab focus even when collapsed
Start tabbing on the page. Right after focussing the URL in the topbar, the focus seems to go nowhere. But it's actually focussing on the hidden tools menu options.

The tools menu should be focus-able, and the options only traversed if the menu is opened.

From: https://bugs.chromium.org/p/chromium/issues/detail?id=971536

### PR DESCRIPTION
**Pull request by Hoten**
Fixes #9168.

### CODE REVIEW

File: lighthouse-core/report/html/templates.html [hunk 1]
```diff
@@ -366,12 +372,12 @@
     <a href="" class="lh-topbar__url" target="_blank" rel="noopener"></a>
 
     <div class="lh-tools">
-      <div class="report-icon report-icon--share lh-tools__button" title="Export report">
+      <button class="report-icon report-icon--share lh-tools__button" title="Export report">
```
Comment (Hoten) [inline lighthouse-core/report/html/templates.html]: If this stays `div`, would have to 1) use `tabindex="0"` and 2) wire our own key event listener so space/enter would toggle the tools menu.

File: lighthouse-core/report/html/templates.html [hunk 2]
```diff
@@ -262,6 +262,14 @@
       height: var(--lh-tools-icon-size);
       cursor: pointer;
       margin-right: 5px;
+      /* Override default button styles. */
```
Comment (patrickhulce) [inline lighthouse-core/report/html/templates.html]: maybe add to the comment that this is an actual `button` element but we want to style it like a transparent `div`? :)

File: lighthouse-core/report/html/templates.html [hunk 3]
```diff
@@ -278,13 +286,10 @@
...
       filter: invert(1);
     }
-    .lh-tools__button:focus,
-    .lh-tools__button.active {
-      outline: none;
-    }
     .lh-tools__button.active + .lh-tools__dropdown {
       opacity: 1;
       clip: rect(-1px, 187px, 242px, -3px);
+      visibility: visible;
```
Comment (patrickhulce) [inline lighthouse-core/report/html/templates.html]: anything extra this does for us compared to opacity?

Comment (Hoten) [reply lighthouse-core/report/html/templates.html]: It removes the elements from focus consideration.

**Comment by brendankenny** [comment]
> Not 100% fitting the accessbility rules of a dropdown

...can you be more specific? :)

**Comment by wardpeet** [comment]
Oh sorry :) It boils down to adding aria attributes and keyboard access. Tabbing isn't the correct way to handle it if you follow the spec. You should be using arrow keys instead.

More info here:
https://www.w3.org/TR/2019/NOTE-wai-aria-practices-1.1-20190207/#menubutton
https://www.w3.org/TR/2019/NOTE-wai-aria-practices-1.1-20190207/examples/menu-button/menu-button-links.html

I don't think it's worth the effort in this PR 👍

**Comment by brendankenny** [comment]
Ah, ok. We should fix that, though :)

**Pull request merged**